config.js

```diff
@@ -53,30 +52,27 @@ config = Object.assign({}, config, {
   // ----------------------------------
   // Compiler Configuration
   // ----------------------------------
-  compiler_devtool: __DEV__ && 'cheap-source-map'
-  || __TEST__ && 'cheap-source-map'
```

Comment:
We use `karma-webpack-with-fast-source-maps` which allows for [faster file sourcemaps](https://github.com/aaronjensen/karma-webpack#file-faster).  Its main trick is that it only re-runs tests that have changed since the last run.  With vanilla `karma-webpack`, if you change one test suite, all 7k+ tests will re-run.

I'm open to changes here but I'd like to address the issue of too many tests taking too long to run while we're at it, or stick with `karma-webpack-with-fast-source-maps`.  Another option I've been considering is using something like the `dot` reporter to lighten up and speed up test output.